src/theme/MarkdownStyler.module.css

```diff
@@ -28,9 +28,5 @@
 }
 
 .markdownStyler a {
-  color: blue;
-}
-
-.markdownStyler a:hover {
-  text-decoration: underline;
+  color: #319795;
```

Comment:
add a comment here saying this mirrors brand color and refer to the file name.

also add a comment in links.ts, saying when updating this color, search for all occurrences of this color and change them all.

and why is this color code different than brand.c, the the color specified in links.ts for links?

Reply:
![image](https://github.com/yuanjian-org/app/assets/83266861/d828f850-69ed-44f6-a020-38e7f1c895f7)
It matches the color.brand.c now